sql/core/src/main/scala/org/apache/spark/sql/execution/ShuffledRowRDD.scala

```diff
@@ -58,6 +58,12 @@ case class PartialMapperPartitionSpec(
     startReducerIndex: Int,
     endReducerIndex: Int) extends ShufflePartitionSpec
 
+// Follow-up: consider using map location for coalescing mappers
```

Comment:
If we plan to leave it as a followup, we should file a JIRA, and fix the comment as, for example, `// TODO(SPARK-XXXXX): blah blah blah`

Reply:
I guess @cloud-fan wants to address https://github.com/apache/spark/pull/33310/files#r669399284 in this PR though (?). cc @maryannxue FYI too.

Reply:
This is different from what I asked, but seems more reasonable. It should be better to read the entire mapper in one task if we need small parallelism, to avoid network data transfer. Except for location, we should also consider the shuffle block size.

Reply:
👌